connectors/llm/interface.py

```diff
@@ -38,28 +43,55 @@ def ask(self, system_prompt, previous_messages, question, agent_id, stream):
                 if "title" in metadata:
                     title = metadata["title"]
                     context_text += f", document title: '{title}'"
-                context_text += ">>\n\n" f"{page_content}\n\n" f"<<Search result {i+1} END>>\n"
+                context_text += (">>\n\n" f"{page_content}\n\n" f"<<Search result {i+1} END>>\n")
 
         prompt = ChatPromptTemplate.from_template(cfg.USER_PROMPT_TEMPLATE)
         prompt_params = {"context": context_text, "question": question}
         log.debug("search result: %s", context_text)
 
+        text_splitter = MistralTokenizer.v3(is_tekken=True)
```

Comment:
Should this variable be named `tokenizer` so that this doesn't become confused with [other text splitters in the code base ](https://github.com/RedHatInsights/tangerine-backend/blob/main/connectors/db/vector.py#L8)?